unit_tests/kernels/UnitTestKernelUtils.h

```diff
@@ -536,6 +540,7 @@ class MixtureFractionKernelHex8Mesh : public TestKernelHex8Mesh
   }
 
   ScalarFieldType* mixFraction_{nullptr};
+  ScalarFieldType* scalarQ_{nullptr};
```

Comment:
`mixFraction_` is the `scalarQ_` that we should be using. No need to introduce another field here.

Reply:
I needed to make it 2 states, so I made another field to make sure I wouldn't break anything else. But I can certainly make mixFraction be 2 states instead.